pinot-core/src/main/java/com/linkedin/pinot/core/segment/creator/impl/SegmentIndexCreationDriverImpl.java

```diff
@@ -535,16 +547,14 @@ void buildIndexCreationInfo()
   @Override
   /**
    * Returns the name of the segment associated with this index creation driver.
-   */
-  public String getSegmentName() {
+   */ public String getSegmentName() {
     return segmentName;
   }
 
   @Override
   /**
    * Returns the path of the output directory
-   */
-  public File getOutputDirectory() {
+   */ public File getOutputDirectory() {
```

Comment:
Ok, figured out why this happened. Since the @Override annotation is above the javadoc, auto formatting in Intellij did that. Fixed by moving the annotation below javadoc.